astroquery/oac/core.py

```diff
@@ -488,12 +488,12 @@ def _parse_result(self, response, verbose=False):
             output_response = self._format_output(raw_output)
 
         except AttributeError:
-            log.debug("ERROR: The web service returned error code: %s" %
+            log.info("ERROR: The web service returned error code: %s" %
                   response.status_code)
             return
```

Comment:
I'm not sure why this, or the KeyError below are not raised. @guillochon or @pcowpert could you please comment on it?

Reply:
These do look like they should be `raises` rather than `returns` @bsipocz. Fine to change them to be I think.

Reply:
So should i `raise` or make a new issue for this?

Reply:
better change it in this PR.

Reply:
Done